client/src/components/Hamburger/Hamburger.css

```diff
@@ -0,0 +1,18 @@
+.hamburger-menu {
+  background-color: white;
+  width: 40px;
+  height: 40px;
+  text-align: center;
+}
+
+.hamburger-menu:hover {
+  cursor: pointer;
+}
```

Comment:
If you make it an `a` tag, you can get rid of this

Reply:
@ZakkMan I switched the tag to an anchor tag and commented out the css code for hover like you requested but the cursor does not change so I left the code in for now.